README.md

```diff
@@ -36,90 +31,10 @@ scarb cairo-lint
 scarb cairo-lint --fix
 ```
 
-Note: You can also include test files with the `--test` flag
-
-## Contributors
```

Comment:
I think we also want to delete https://github.com/software-mansion/cairo-lint/blob/main/.all-contributorsrc